source/install.txt

```diff
@@ -71,8 +71,33 @@ Procedure
 
    .. tab::
       :tabid: linux
+
+      Select the appropriate tab based on your Linux distribution and
+      desired package:
+
+      - For TGZ tarball installation, click the ``.tgz`` tab.
+
+      - For installation on Ubuntu and Debian, click the ``.deb`` tab.
+
+      - For installation on :abbr:`RHEL (Red Hat Enterprise Linux)`,
+        click the ``.rpm`` tab.
```

Comment:
• Nice. Only thought is that you can install the TGZ on Ubuntu and Debian and RHEL. To further differentiate, how about:

>Select the appropriate tab below based on your installation method:
> 
>- To install via the RPM package on RHEL, click ...
>- To install via the DEB package on Ubuntu and Debian, click ...
>- To install using the TGZ tarball, click ...

Reply:
• "below" helps because they are now surrounded by tabs! This helps them to focus their attention on the right tabs for the next step.

• I generally leave TGZ steps to the end (i.e. as the third tab, here), but it's up to you. Ideally, most users will use the package manager steps.